charts/celestia-node/README.md

```diff
@@ -1,6 +1,6 @@
 # celestia-node
 
-![Version: 0.10.3](https://img.shields.io/badge/Version-0.10.3-informational?style=flat-square) ![AppVersion: v0.20.1](https://img.shields.io/badge/AppVersion-v0.20.1-informational?style=flat-square)
+![Version: 0.10.5](https://img.shields.io/badge/Version-0.10.5-informational?style=flat-square) ![AppVersion: v0.20.2](https://img.shields.io/badge/AppVersion-v0.20.2-informational?style=flat-square)
```

Comment:
node is on 20.4 right?

Reply:
can this be auto updated with the scripts maybe? since we would be setting the versions